Bottom: 396354cd61f09ba6d1e3feace66592bb4bf857d9
Top:    366c7831dd7fb4941504754c2759441128ea1e1e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:58:18 +0100

action.c: Remove unused action "MovePointer"


---

diff --git a/src/action.c b/src/action.c
index 6339e39..704b98f 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2108,25 +2108,6 @@ ActionSetThermal (int argc, char **argv, int x, int y)
 }
 
 /* ---------------------------------------------------------------------------
- * action routine to move the X pointer relative to the current position
- * syntax: MovePointer(deltax,deltay)
- */
-void
-ActionMovePointer (char *deltax, char *deltay)
-{
-  LocationType dx, dy;
-
-  dx = (LocationType) (atoi (deltax) * PCB->Grid);
-  dy = (LocationType) (atoi (deltay) * PCB->Grid);
-
-  notify_crosshair_change (false);
-  MoveCrosshairRelative (TO_SCREEN_SIGN_X (dx), TO_SCREEN_SIGN_Y (dy));
-  /* update object position and cursor location */
-  AdjustAttachedObjects ();
-  notify_crosshair_change (true);
-}
-
-/* ---------------------------------------------------------------------------
  * !!! no action routine !!!
  *
  * event handler to set the cursor according to the X pointer position
diff --git a/src/action.h b/src/action.h
index ee116e8..e92c578 100644
--- a/src/action.h
+++ b/src/action.h
@@ -35,7 +35,6 @@
 
 #define CLONE_TYPES LINE_TYPE | ARC_TYPE | VIA_TYPE | POLYGON_TYPE
 
-void ActionMovePointer (char *, char *);
 void ActionAdjustStyle (char *);
 void EventMoveCrosshair (int, int);
